active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,225 @@
+V2 will add this. This spec focuses on V1 Breadcrumb,
+but keeping in mind that V1 is positioned in a way to have V2 features smoothly added later. 
+
+# API Pages
+
+## Breadcrumb class
+
+Represents a control that shows a list of items, for example the navigation trail to the current location.
+
+### Examples
+
+The following example code creates a simple Breadcrumb control with three nodes.
+When there is not enough width to display the whole control,
+the leading nodes are replaced with a node that the user can click on,
+opening the missing nodes in a flyout menu.
+
+![Breadcrumb with nodes: Home, Documents, Design, Northwind, Images. The app is resized so that the Breadcrumb crumbles - then, clicking the ellipsis reveals the crumbled Home and Documents nodes](images/Breadcrumb_crumbling_flyout.gif)
+
+XAML
+```XAML
+<Breadcrumb x:Name="Breadcrumb1" /> 
+```
+
+C#
+```cs
+Breadcrumb1.ItemsSource = new string[]{ "Home", "Documents", "Design", "Northwind", "Images"}
+```
+
+If the last item in a Breadcrumb is text and is too long to fit it will be clipped
+(if the item type is a string it will be trimmed, meaning shortened and replaced with an ellipsis).
+
+![Breadcrumb_crumbled with last node truncated](images/Breadcrumb_truncation.PNG)
+
+#### Responding to clicks
+
+This example shows an app with a Breadcrumb across the top, and a content area on the rest of the Window.
+The Breadcrumb shows a list of items. Clicking on an item navigates the content back, and updates
+the Breadcrumb.
+
+```xaml
+<Grid RowDefinitions="Auto,*">
+    <Breadcrumb ItemClicked="CrumbClicked" ItemsSource="{x:Bind Items, Mode=OneWay}" />
+    
+    <ContentControl Grid.Row="1" Content="{x:Bind CurrentItem, Mode=OneWay}">
+        ...
+    </ContentControl>
+</Grid>
+```
+
+```csharp
+public sealed partial class MainPage : Page
+{
+    public ObservableCollection<Items> Items { get; }
+
+    public ItemClickEventArgs Current Item { get; set;}
+
+    void CrumbClicked(Breadcrumb sender, BreadcrumbItemClickedEventArgs e)
+    {
+        for(int i = Items.Count-1; i > e.Index; i++)
+        {
+            Items.RemoveAt(i);
+        }
+
+        CurrentItem = Items[i - 1];
+    }
+
+    void NavigateToItem(Item item)
+    {
+        Items.Add(item);
+        CurrentItem = item;
+    }
+}
+```
+
+#### Breadcrumb.ItemTemplate property
+
+Gets or sets the template to display an item
+
+The Breadcrumb ItemTemplate is used to override the default display of the
+individual items in the Breadcrumb control.
+
+The following uses the ItemTemplate so that the Breadcrumb items show the Name property of the items:
+
+```xaml
+<Breadcrumb ItemsSource='{x:Bind Items}'>
+    <Breadcrumb.ItemTemplate>
+        <DataTemplate x:DataType='BreadcrumbItem`>
+            <TextBlock Text='{x:Bind Name}/>
+        </DataTemplate>
+    </Breadcrumb.ItemTemplate>
+</Breadcrumb>
+```
+
+Notes:
+* You can use a BreadcrumbItem as the root of the DataTemplate
```

Comment:
> * You can use a BreadcrumbItem as the root of the DataTemplate

How will this be different to the example given above?
If not using a BreadcrumbItem as the root of the data template is what's provided set as the `Content` of a BreadcrumbItem automatically?
What if I tried to nest Breadcrumb controls by setting a `Breadcrumb` in the ItemTemplate of another Breadcrumb? I'm assuming it won't be a supported scenario but could it have implications that could or should be detected and reported as a warning or error?